File: Heap Dynamic/HD.java
import java.util.ArrayList;

public class HD {
    public static void main(String[] args) {
        ArrayList<Integer> arr = new ArrayList<>();
        arr.add(10);
        arr.add(20);
        arr.add(30);
        arr.add(1, 15);
        System.out.println("\nHeap-Dynamic Array (ArrayList):");
        for (int num : arr) {
            System.out.print(num + " ");
        }
        arr.remove(2);
        System.out.println("\nAfter removal:");
        for (int num : arr) {
            System.out.print(num + " ");
        }
    }
}

File: Heap Dynamic/HD.py
def heap_dynamic():
    arr = []
    arr.append(10)
    arr.append(20)
    arr.append(30)
    arr.insert(1, 15)
    print("\nHeap-Dynamic Array (Python list):")
    print(arr)
    arr.pop(2)
    print("After removal:")
    print(arr)
heap_dynamic()